instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/GrpcRpcAttributesGetter.java

```diff
@@ -37,4 +41,24 @@ public String method(GrpcRequest request) {
     }
     return fullMethodName.substring(slashIndex + 1);
   }
+
+  @Nullable
+  public List<String> metadataValue(GrpcRequest request, String key) {
```

Comment:
Nit: does not need to be public
```suggestion
  List<String> metadataValue(GrpcRequest request, String key) {
```

Reply:
So this is not available outside the package, I agree